### PR TITLE
Cdn update

### PR DESCRIPTION
**Pull request by SpencerMcFadden**

### CODE REVIEW

File: lss-profile-picture.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
                 cursor: pointer;
             }
         </style>
-        <img draggable="false" src$="[[src]]"></img>
+        <img id="profileImage" draggable="false" src$="[[src]]"></img>
```
Comment (antwonthegreat) [inline lss-profile-picture.html:26]: There are other places in the app where profile pictures are used directly as img tags (not using lss-profile-picture) like home-profile.html line 199.

Not sure but searching for Default.Picture may find all of these

File: src/lss-profile-picture.ts [hunk 2]
```diff
@@ -40,7 +40,11 @@ class LSSProfilePicture extends Polymer.Element {
...
   @Polymer.decorators.observe('personId,size')
   getSrc(personId: number, size: number) {
-    const baseUrl = this.isDev() ? 'https://devmapi.leavitt.com/' : 'https://mapi.leavitt.com/';
-    this.src = `${baseUrl}People(${personId})/Default.Picture(size=${size})`;
+    this.src = `https://cdn.leavitt.com/user-${personId}-${size}.jpeg`;
+  }
+
+  @Polymer.decorators.listen('error', 'profileImage')
+  onProfilePictureError(size: number) {
+    this.src = `https://cdn.leavitt.com/user-0-${size}.jpeg`;
```
Comment (aarondrabeck) [inline src/lss-profile-picture.ts:48]: We need to intermediate prop here.  If there where to be an error getting the default image, a loop would be created.

File: lss-profile-picture.html (hunk 1 above)
Comment (SpencerMcFadden) [reply lss-profile-picture.html:26]: On home yes. This is the update for the profile picture and menu package that is used on home.

File: src/lss-profile-picture.ts (hunk 2 above)
Comment (aarondrabeck) [reply src/lss-profile-picture.ts:48]: This component need to get the next closes size too.

Comment (SpencerMcFadden) [reply src/lss-profile-picture.ts]: Added those changes

**Pull request merged**